src/main/java/org/commonjava/indy/service/archive/model/dto/HistoricalContentDTO.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2011-2021 Red Hat, Inc. (https://github.com/Commonjava/indy)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.service.archive.model.dto;
+
+import java.util.Set;
+
+public class HistoricalContentDTO
+{
+
+    private Set<HistoricalContentEntryDTO> downloads;
+
```

Comment:
Not sure if uploads are not needed here or just missed.

Reply:
This is used to do obj mapper for the list PNC provided, if sidecar will redirect all maven uploads to remote Indy, I see uploads are not needed in archive generation.